util/p9/replace

```diff
@@ -0,0 +1,14 @@
+#! /bin/sh
+
+timestamp=$(date +"%Y-%m-%d_%H-%M-%S")
+
+echo "removing gatling-puppet-load-test"
+rm -rf ./gatling-puppet-load-test
+
+echo "cloning gatling-puppet-load-test"
+git clone https://github.com/puppetlabs/gatling-puppet-load-test.git
+
+echo "refresh complete; adding timestamp to gatling-puppet-load-test"
+touch ./gatling-puppet-load-test/${timestamp}.txt
```

Comment:
If a target destination path is added above, this line will need to be updated.

Reply:
Added [docs/slv_environment_setup.md](https://github.com/puppetlabs/gatling-puppet-load-test/blob/SLV-321/docs/slv_environment_setup.md) with a [Utility scripts](https://github.com/puppetlabs/gatling-puppet-load-test/blob/SLV-321/docs/slv_environment_setup.md#utility-scripts) section.